flow/table.go

```diff
@@ -271,6 +275,14 @@ func (ft *Table) update(updateFrom, updateTime int64) {
 		}
 
 		f.XXX_state.lastMetric = f.Metric.Copy()
+
+		if f.FinishType != FlowFinishType_NOT_FINISHED {
+			if updateTime-f.Last >= HoldTimeoutMilliseconds {
```

Comment:
We just need to do that

```
			if updateTime-f.Last >= HoldTimeoutMilliseconds {
				delete(ft.table, k)
			}

```
We don't need to check if any packet arrvied in the flow 15sec after, as we tested already line 280 that the flow didn't received any packets.
So you can remove the HoldTimeout ticker mechanism and holdFlows map

Reply:
I don't understand.
If I remove the holdticker then flows will only get deleted after "flow.update" - 60 seconds, instead of 15 seconds.
This is what we had before,  but I added the holdticker as @hunchback and @safchain suggested.

Reply:
@orozery It's a bit confusing I understand, but as discuss with @safchain ; The whole purpose of this feature didn't require a ticker, basically you just need to expire the flow (with timeout flag and no packets arrived last HoldTimeout).
This will have same behavior and will avoid over engineering.